### DISCUSSION TITLE
Is there any config to control the default behavior of "show more" in completion items?

### DISCUSSION THREAD

**Discussion by HaoboGu** [eclipse-theia/theia#7859] [General] [answered]
### Feature Description:
Hello, I am an extension developer. I am writing a completion extension for theia. I found that we I start a theia ide, the details/documents view of completion item will not be shown by default:
![image](https://user-images.githubusercontent.com/8640918/82516583-bf26ce80-9b4d-11ea-8da8-37c35735094d.png)

Is there any configuration I can change to show to full information of completion items? 
Thanks!

**Comment by vince-fugnitto** [comment, marked as answer]
@HaoboGu I transferred the issue to **discussions** since it is a question.
As far as I can tell there is no preference which controls the behaviour of showing the quick info (the additional information) during suggestions or auto-completion. In addition, I was unable to find a preference for it in vscode either, and we are reusing the ‘Monaco Editor’ which ultimately implements this feature. There is a keybinding however to trigger additional information if end-users which to see it.

**Reply by HaoboGu** [reply]
Thanks for the reply! 

If I want this feature, where should I propose this feature request?

**Reply by vince-fugnitto** [reply]
> Thanks for the reply!
> If I want this feature, where should I propose this feature request?

I believe you can open the issue directly in vscode (they will delegate the issue to monaco editor if required). I do notice that when choosing to display additional information, vscode persists this choice, while theia may not (which can be improvement to be made on our side).
